wazimap_ng/profile/serializers/indicator_data_serializer.py

```diff
@@ -11,10 +12,13 @@
 logger = logging.getLogger(__name__)
 
 def get_indicator_data(profile, geography):
+    if not isinstance(geography, Iterable):
```

Comment:
wouldn't it be better to just make this accept a list of things. 
so `geographies` and in 
https://github.com/OpenUpSA/wazimap-ng/blob/3a38bdfb5ea595d1220d0a2667224ec78a29b6cc/wazimap_ng/profile/serializers/indicator_data_serializer.py#L92

use `indicator_data = get_indicator_data(profile, [geography]) `